web/ui/src/Services/servicesFactory.js

```diff
@@ -29,7 +29,7 @@
         angular.extend(newFactory, {
             // TODO - update list by application instead
             // of all services ever?
-            update: function(force){
+            update: function(force, skipUpdateInstances){
```

Comment:
add flag to skip updating instances because for services map we want to make that request directly so we can wait on it.